src/sass/app.scss

```diff
@@ -88,80 +109,94 @@ a {
 	padding: 1px 8px;
 }
 
-.panel-notificacion {
+.notification-panel {
 	@include display-flex();
 	@include align-items(center);
 	background-color: #ffffff;
-	width: 50%;
 	border-radius: 8px;
 	padding: 22px;
 	margin: 35px 0px;
 }
 
-.punto-notificacion {
-	@include align-self(start);
-	background-color: #FF2861;
-	width: $anchopunto;
-	height: 16px;
+.image-container { 
+	display: flex;
+	@include flex($width-image-container);
+	position: relative;
+}
+
+.notification-badge {
+	position: absolute;
```

Comment:
@cuartas15 all elements absolutely positioned MUST have the vertical and horizontal positions (`top - left or bottom - right`) positions, it works well on chrome but we don't know how it will perform on other browsers:
![image](https://user-images.githubusercontent.com/662930/38461992-598e591e-3aa4-11e8-82b5-24e82d28f067.png)